README.md

```diff
@@ -162,4 +172,136 @@ $ oc get image | grep onload
 ```
 Remove any outstanding manually with `oc delete image`. (Not providing any automated invocation to prevent removal of the false-positive images.)
 
+## Deploying artifacts into an airgapped cluster
+
+### Images required
+
+This is a list of the images that are currently needed as reported by podman.
```

Comment:
```suggestion
This is a list of the images that are currently needed as reported by podman:
```

I know it's really nit-picking but I can't ignore typography inconsistency! We tend to use : (colon mark) for referring to the code snippets. If you disagree, then choose any approach and stick to it here and elsewhere like "Podman login". 👍